javascript/javascript1/week2/homework-week2/candy-helper.js

```diff
@@ -0,0 +1,41 @@
+const boughtCandyPrices = [];
+const amountToSpend = Math.random() * 100;
```

Comment:
Optional Challenge
This returns a random number with many decimals. You could find away to return only integer.